server/Dockerfile

```
@@ -24,7 +24,8 @@ RUN adduser -D -u 1000 -G root -h /mesh mesh && \
     mkdir /mesh/data && \
     mkdir -p /mesh/elasticsearch/data && \
     chown 1000:0 /mesh -R && \
-    chmod 770 /mesh -R
+    chmod 770 /mesh -R && \
+    apk update && apk add rsync
```

I suggest to remove the apk cache after the adding the rsync program to keep the image size small
```suggestion
    apk update && apk add rsync && rm -rf /var/cache/apk/*
```